.travis.yml

```diff
@@ -176,3 +177,18 @@ jobs:
       env:
         - MPLBACKEND=ps
         - PYTHON_VERSION=3.7.2
+
+    # Randomized testing and benchmarks
+    - stage: randomized
+      name: Randomized tests
+      <<: *stage_linux
+      language: python
+      script:
+      - python setup.py build_ext --inplace
+      - make test_randomized
+      python: 3.5
```

Comment:
This can also be dropped, it's already defined on: https://github.com/Qiskit/qiskit-terra/pull/2620/files#diff-354f30a63fb0907d4ad57269548329e3R52

Reply:
:+1: removed both in 38caf0c235c57bc6e81af0b499ef4c95e05152ec